### PR TITLE
restructure and simplify the template

### PR DESCRIPTION
**Pull request by edgarriba**
restructure the template project following the structure below

```bash
repository-name/  # Root level of the project.
├── libs/ # Contains private libraries.
|   └── project_name/
|       ├── ops.py  # e.g a operators submodule
|       └── project_name_subpackage/
|           └── utils.py  # e.g a utilites submodule
├── src/ # Contains all code needed to run the main gui application.
|   ├── assets/  # Contains files needed to run the application
|   |   └── app_logo.png  # static images or images for buttons.
|   ├── res/  # Contains the layout files and UI strings.
|   |   └── main.kv  # main ui layout in Kivy languague
|   └── main.py  # Is the main entry point for the gui application.
├── test/  # Contains code for test of the private libs.
|   └── test_dummy.py  # sets of unit test.
├── entry.sh
|    # The script to setup the project, create a virtual env. and install dependencies.
└── setup.cfg
    # The file containing the metadata of the package, including the name, versioning, etc. Learn more here: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html

```

- TODO: add small ci

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,16 +4,15 @@ requires = [
     "certifi==2022.9.24",
     "charset-normalizer==2.1.1",
     "docutils==0.19",
+    "farm-ng-amiga",
```
Comment (edgarriba) [inline pyproject.toml:7]: @christopherbradski the reset of the list should be review, what is really needed and not

File: README.md [hunk 2]
```diff
@@ -36,36 +19,77 @@ Fill in the details, example below:
...
-3. enter your chosen app name as well.
-4. Press enter again
-5. Press enter once more to accept the formatted package name
-    - Or rename the package as you desire
+In both cases you will have to clone the created repository from above
 
-Your kivy application has been created under the *apps* directory
 ```bash
-ls apps/
+git clone https://github.com/edgarriba/amiga-kornia-app.git
```
Comment (Hackerman342) [inline README.md:30]: Should make it clear it is their repo they clone here

**Review by ethanrublee — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -36,36 +19,77 @@ Fill in the details, example below:
...
+        The file containing the metadata of the package, including the name, versioning, etc. Learn more here: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
+
+
+## How to setup the project
 
 Before any changes, lets see if we can run this app locally on your system.
+
 ```bash
-apps/<your app name>/entry.sh
+cd amiga-kornia-app/
```
Comment (Hackerman342) [inline README.md:67]: same here about it being their app

File: README.md [hunk 4]
```diff
@@ -36,36 +19,77 @@ Fill in the details, example below:
...
 
-If all goes well, you'll see an empty kivy application on your screen.
+
+## Customizing and Debug an app
+
+The workflow for development is pretty much the same as any standard gui application.
+
+1. Make changes in the code.
+2. Run the code with the play button in vs-code.
+2.1. [Optionally] Add a breakpoint to any line and use the Debug Console to interact.
```
Comment (Hackerman342) [inline README.md:82]:
```suggestion
  - [Optionally] Add a breakpoint to any line and use the Debug Console to interact.
```

File: README.md [hunk 5]
```diff
@@ -36,36 +19,77 @@ Fill in the details, example below:
...
+3. Go to step 1)
+
+
+## Test your application
```
Comment (Hackerman342) [inline README.md:86]:
```suggestion
### [Optional] Create tests for your application
```

File: libs/amiga_pkg/math.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+"""Template module with math functions."""
+
```
Comment (Hackerman342) [inline libs/amiga_pkg/math.py:2]: Maybe we can rename? I foresee less experienced programmers running into issues with this and python's math module

File: README.md [hunk 7]
```diff
@@ -36,36 +19,77 @@ Fill in the details, example below:
...
+        `assets/`
+            Contains files needed to run the application. e.g. static images for buttons.
+        `res/`
+            Contains the layout files and UI strings, e.g in the Kivy language.
+    `test/`
+        Contains code for test of the private libs.
+    `entry.sh`
+        The script to setup the project, create a virtual env. and install dependencies.
+    `setup.cfg`
+        The file containing the metadata of the package, including the name, versioning, etc. Learn more here: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
```
Comment (Hackerman342) [inline README.md:59]: Add bullets to properly format

File: setup.cfg [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# NOTE: update the `[project]` section with your app details
+[project]
```
Comment (Hackerman342) [inline setup.cfg:2]: Should have instructions in readme / docs to change these metadata values

**Review by Hackerman342 — APPROVED** [review]
Nice, especially including using a `.kv` file in the example. 

Just some doc nits in the review

**Pull request merged**